python/composio/tools/env/filemanager/manager.py

```diff
@@ -191,6 +191,7 @@ def grep(
         pattern: t.Optional[t.Union[str, Path]] = None,
         recursive: bool = True,
         case_insensitive: bool = True,
+        exclude: t.Optional[t.List[t.Union[str, Path]]] = None,
```

Comment:
Consider updating the type hint for the 'exclude' parameter to use `t.Optional[t.List[str]]` instead of `t.Optional[t.List[t.Union[str, Path]]]`. This would be more consistent with how the exclude paths are used in the implementation.